deployment/terraform/azure/container_app/variables.tf

```diff
@@ -0,0 +1,160 @@
+variable "event_listener" {
+  type = object({
+    type = string
+
+    # POLLING
+    resync_on_start = optional(bool)
+    interval        = optional(number)
+
+    # WEBHOOK
+    app_host = optional(string)
+
+
+    # KAFKA
+    brokers                  = optional(list(string))
+    security_protocol        = optional(list(string))
+    authentication_mechanism = optional(list(string))
+    kafka_security_enabled   = optional(list(bool))
+    consumer_poll_timeout    = optional(list(number))
+  })
+
+  default = {
+    type = "POLLING"
+  }
+  description = "The event listener configuration"
+}
+
+variable "additional_secrets" {
+  type = map(string)
+  default = {}
+  description = "Additional secrets to be injected into the container, the key of the map is the name of the secret and will be the environment variable name"
```

Comment:
This is not clear enough, need a better example IMO. Is this strictly key value?